app/views/shelters/show.html.erb

```diff
@@ -11,3 +11,15 @@
 </div>
 
 <h2><%= link_to "Shelter's Pets", "/shelters/#{@shelter.id}/pets" %></h2>
+
+<h2>Read Our Reviews!</h2>
+<% @reviews.each do |review| %>
```

Comment:
Linking back up to my previous message. You could do something like `@shelter.review.each`